ReactNativeClient/lib/components/screens/config.js

```diff
@@ -30,11 +31,48 @@ class ConfigScreenComponent extends BaseScreenComponent {
 			await shared.checkSyncConfig(this, this.state.settings);
 		}
 
-		this.saveButton_press = () => {
+		this.saveButton_press = async () => {
+			if (
+				this.state.changedSettingKeys.includes('sync.target')
+				&& this.state.settings['sync.target'] === SyncTargetRegistry.nameToId('filesystem')
+				&& !await this.checkFilesystemPermission()
+			) {
+				Alert.alert(
+					_('Permission error'),
+					_(
```

Comment:
Please put this string on one line, as otherwise I don't think it will be picked up by the translation software.

Reply:
Do you use `xgettext` for extracting strings? I just tried and it gave me correct result:
```
$ xgettext --version
xgettext (GNU gettext-tools) 0.19.8.1
$ xgettext lib/components/screens/config.js --output=test.po --language=JavaScript --no-location
```

```po
#: lib/components/screens/config.js:43
msgid ""
"Filesystem permission denied. Please choose another synchronization target."
msgstr ""
```